crates/miden-lib/asm/kernels/transaction/lib/asset_vault.masm

```diff
@@ -29,7 +29,7 @@ const.ERR_VAULT_NON_FUNGIBLE_ASSET_TO_REMOVE_NOT_FOUND="failed to remove non-exi
 # EVENTS
 # =================================================================================================
 
-const.SMT_PEEK=event("stdlib::collections::smt::smt_peek")
+const.SMT_PEEK_EVENT=event("stdlib::collections::smt::smt_peek")
```

Comment:
nit: I would probably add a comment that this event is actually handled by the standard library handlers, and we are kind of "hijacking" it here. But we should be careful to make sure the event names don't drift apart (maybe there could be some way to check this in the future).